entrypoint.sh

```diff
@@ -1,56 +1,20 @@
 #!/bin/bash
-# shellcheck disable=SC2046,SC2001,SC2086
 set -e
 
-# Pass User-Agent Inforomation to the 1Password CLI
-export OP_INTEGRATION_NAME="1Password GitHub Action"
-export OP_INTEGRATION_ID="GHA"
-export OP_INTEGRATION_BUILDNUMBER="1010001"
-
-readonly CONNECT="CONNECT"
-readonly SERVICE_ACCOUNT="SERVICE_ACCOUNT"
-
-auth_type=$CONNECT
-managed_variables_var="OP_MANAGED_VARIABLES"
-IFS=','
-
-if [[ "$OP_CONNECT_HOST" != "http://"* ]] && [[ "$OP_CONNECT_HOST" != "https://"* ]]; then
-  export OP_CONNECT_HOST="http://"$OP_CONNECT_HOST
-fi
-
-# Unset all secrets managed by 1Password if `unset-previous` is set.
-unset_prev_secrets() {
-  if [ "$INPUT_UNSET_PREVIOUS" == "true" ]; then
-    echo "Unsetting previous values..."
-
-    # Find environment variables that are managed by 1Password.
-    for env_var in "${managed_variables[@]}"; do
-      echo "Unsetting $env_var"
-      unset $env_var
-
-      echo "$env_var=" >> $GITHUB_ENV
-
-      # Keep the masks, just in case.
-    done
-
-    managed_variables=()
-  fi
-}
-
 # Install op-cli
 install_op_cli() {
+  CLI_VERSION="v2.16.0-beta.01"
```

Comment:
You could check out what powers `op update` to see how it checks for the latest CLI version. That said I don't mind hard-coding the version. Despite our best efforts we could introduce regressions or even intentionally have a breaking change in future CLI versions and break folks workflows.

Reply:
Question: is there a use case setting the version is an input to let the user control it?

Reply:
> Is there any way we could get the most recent 1Password CLI version number programmatically?
Yes, there is, and I am switching to that one in this PR since it's one line of code 😄 

> You could check out what powers `op update` to see how it checks for the latest CLI version.
This was a great suggestion which lead to the elegant solution for this.

> That said I don't mind hard-coding the version. Despite our best efforts we could introduce regressions or even intentionally have a breaking change in future CLI versions and break folks workflows.
I'm planning on working on a separate GitHub Action that will install the CLI. In that one, getting the latest version will be the default. Users can specify which version they want to use if they want to manually update the CLI in their pipelines.

> Is there a use case setting the version as an input to let the user control it?
Yes, there is. The simplest example is ensuring that no surprises happen in the pipeline by accidentally getting the latest version of the CLI that turns out to break a pipeline. Another use case is to make sure that the CLI behaves as expected. We may improve things in future releases that may change some behavior (for the better) and some folks may want to double check that it doesn't affect their pipelines before updaing the CLI.

In the install CLi GitHub Action that I'm working on, folks will be able to specify the version they want to use. This comes in handy especially if they want to access beta features (which is another use case).

Reply:
And the change that will programatically get the latest version of the CLI is found in commit [`17a2a72`](https://github.com/1Password/load-secrets-action/pull/36/commits/17a2a72e87dddbb35f43825ef9f56a930693f654).